### DISCUSSION TITLE
Updated SpiralLinux to Debian 12

### DISCUSSION THREAD

**Discussion by P1tape** [SpiralLinux/SpiralLinux-project#267] [General]
I have upgraded SpiralLiux to Debian 12 in VirtualBox (following instructions on the Wiki). So far very smooth and no obvious bugs encountered. Below the system info.

![VirtualBox_Spiral_11_06_2023_01_05_09](https://github.com/SpiralLinux/SpiralLinux-project/assets/134995867/dbef8b4d-ee5a-4c13-ae9c-fdf7f4251aeb)

**Comment by geckolinux** [comment]
Thanks a lot for the report. Did Synaptic throw an error like this?

> Repository 'http://deb.debian.org/debian stable InRelease' changed its 'Version' value from '11.7' to '12.0'Repository 'http://deb.debian.org/debian stable InRelease' changed its 'Codename' value from 'bullseye' to 'bookworm'

![image](https://github.com/SpiralLinux/SpiralLinux-project/assets/2692138/343ae820-d440-44e0-9c5c-213bffce43b7)

**Reply by P1tape** [reply]
I did not have such error message. I just followed your wiki instructions to disable and renable repositories.  By the way I am a bit disappointed by the selection of default software in Debian 12. Multiple weird audio or video player, kde settings app in gnome, ugly xfce interface, multiple useless games, low contrast fonts barely readable, etc. Mint LMDE and Spiral are more polished. Keep improving it with Spiral!

**Reply by WeirdGnome** [reply]
Installed Spiral Linux yesterday and after that updated to bookworm. Always liked debian but the default settings for lxqt are crazy. Spiral linux solves this issue =) Thank you for creating such amazing system =) love it!

**Reply by fweller** [reply]
I did not have any issues with upgrading my Spiral Plasma install to Bookworm.

**Reply by fweller** [reply]
@geckolinux as it turns out, there is a small issue that I can see when upgrading SpiralLinux to Bookworm.  On the KDE Plasma flavor, once upgraded to Bookworm, the menu for Settings / Appearance / Splash Screen does not have permissions to access any themes or images.  I have not figure out yet how to fix this.
